content/document.test.js

```diff
@@ -0,0 +1,58 @@
+const fs = require("fs");
+const path = require("path");
+
+const Document = require("./document");
+
+describe("test Document.findAll()", () => {
+  it("should always return files that exist", () => {
+    const filePaths = [...Document.findAll().iter({ pathOnly: true })];
+    expect(filePaths.every((value) => fs.existsSync(value))).toBeTruthy();
```

Comment:
Isn't this just testing whether fdir works? We are never adding to what it finds, so this should be given. But I'm cool with testing it nonetheless, 3 lines for making sure the lib is working as expected is still a good use of space in my book